react/react-training/src/screens/Game.js

```diff
@@ -0,0 +1,93 @@
+import React, { Component } from 'react';
```

Comment:
Change the name of the file to `index.js` because that way you can import it as `import Game from './screens/Game'`